interface/patient_file/report/custom_report.php

```diff
@@ -488,21 +488,21 @@ function next_prev(action){
   // old code used the global custom dir which is no longer a valid
    $practice_logo = "$OE_SITE_DIR/images/practice_logo.png";
    if (file_exists($practice_logo)) {
-        echo "<img src='$practice_logo' align='left'><br />\n";
-     } 
+        echo "<img src='$practice_logo' align='left' height='80' width='48'><br />\n";
+     }
 ?>
-<h2><?php echo $facility['name'] ?></h2>
+<h2>&nbsp;&nbsp;&nbsp;<?php echo $facility['name'] ?></h2>
 <?php echo $facility['street'] ?><br>
 <?php echo $facility['city'] ?>, <?php echo $facility['state'] ?> <?php echo $facility['postal_code'] ?><br clear='all'>
 <?php echo $facility['phone'] ?><br>
 
-<a href="javascript:window.close();"><span class='title'><?php echo $titleres['fname'] . " " . $titleres['lname']; ?></span></a><br>
+<span class='title'><?php echo $titleres['fname'] . " " . $titleres['lname']; ?></span><br>
```

Comment:
Why are you removing the hyperlink to close the screen?

Reply:
@pri2si17-1997 Said that I can do it.

Reply:
But I can restore it, if you want @teryhill

Reply:
Ok @pri2si17-1997 @teryhill 
I'll work on it soon